appveyor.yml

```diff
@@ -1,22 +1,45 @@
-version: "{build}"
-build: off
-skip_tags: true
+build: false
+skip_branch_with_pr: true
 
 environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
     - nodejs_version: "10"
+    - nodejs_version: "Current"
 
+configuration: Release
 platform:
   - x86
   - x64
 
 install:
+  - SET PATH=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin;%PATH%
   - ps: Install-Product node $env:nodejs_version $env:platform
   - set PATH=%APPDATA%\npm;%APPVEYOR_BUILD_FOLDER%\node_modules\.bin;%PATH%
   - git submodule update --init --recursive
   - npm i
 
 test_script:
+  - node --version
+  - npm --version
   - npm test
+
+after_test:
+  - ps: If ($env:nodejs_version -eq "Current") { npm run prebuild }
+
+artifacts:
+  - path: prebuilds
+    name: $(APPVEYOR_REPO_TAG_NAME)-win-$(PLATFORM)
+    type: zip
+
+deploy:
+  - provider: GitHub
+    artifact: /.*\.zip/
+    auth_token:
+      secure: FIXME
```

Comment:
Not sure we can use a env var here; it's not documented. So let's just use https://ci.appveyor.com/tools/encrypt

Reply:
@vweevers Can you take care of this?